src/Motor.Extensions.Hosting.RabbitMQ/RabbitMQMessagePublisher.cs

```diff
@@ -47,16 +47,8 @@ public async Task PublishMessageAsync(MotorCloudEvent<byte[]> motorCloudEvent, C
             properties.DeliveryMode = 2;
             properties.Update(motorCloudEvent, _options);
 
-            var exchange = motorCloudEvent.GetRabbitMQExchange() ?? _options.PublishingTarget.Exchange;
-            var routingKey = motorCloudEvent.GetRabbitMQRoutingKey() ?? _options.PublishingTarget.RoutingKey;
```

Comment:
@baez90 Do you know why we did this in the first place? This behavior was present since the days of GData.Hosting and it doesn't make any sense to me. The `exchange` and `routingKey` of a `MotorCloudEvent` refer to the binding of the queue which it was received from. Why on earth should this binding be used for the publisher, too?

Reply:
The first idea that comes to mind is that this allows to pass the publishing coordinates to the publisher also as metadata from the underlying publisher which in turn allows conditional publishing to multiple targets from a single publisher. I have an idea why this was necessary/useful but I've to dig in a little bit deeper

Reply:
I think we missed that the exchange and routing key are only set if the option `ExtractBindingKey` is set to true. Now it makes a little bit more sense and I start to see possible use cases. @pkemkes will revert that change and only keep the fixed serialization format

Reply:
Sounds good, I assume e.g. the `IMultiOutputService` benefits from the option to override some of these options.
Thanks for checking everything!